Tests/KumoTests/Fixtures/Common/NetworkTest.swift

```diff
@@ -1,56 +1,60 @@
+import Combine
 import Foundation
-import RxSwift
-import XCTest
 @testable import Kumo
+import XCTest
 
 class NetworkTest: XCTestCase {
-    
     let service = Service(baseURL: URL(string: "https://httpbin.org")!)
-    
+
     let parameters: (actual: [String: Any], expected: [String: String]) = {
         let base: [String: Any] = ["foo": 1, "bar": "foo"]
         return (actual: base, expected: base.mapValues(String.init(describing:)))
     }()
-    
-    func successfulTest<T>(of observable: Observable<T>, file: StaticString = #file, line: UInt = #line, function: String = #function) -> (_ description: String) -> ((_ successCondition: @escaping (T) -> Bool) -> ()) {
+
+    func successfulTest<T>(of observable: AnyPublisher<T, Error>, file: StaticString = #file, line: UInt = #line, function: String = #function) -> (_ description: String) -> ((_ successCondition: @escaping (T) -> Bool) -> Void) {
         return { description in
             { successCondition in
                 var emissions = [T]()
                 let expect = self.expectation(description: description)
-                _ = observable
-                    .subscribe(onNext: {
-                        emissions.append($0)
-                    }, onError: { error in
+                observable.sink(receiveCompletion: { completion in
+                    switch completion {
+                    case let .failure(error):
                         XCTFail("Expectation violated - test '\(function)' emitted an error: \(error).", file: file, line: line)
                         expect.fulfill()
-                    }, onCompleted: {
+                    case .finished:
                         XCTAssertTrue(emissions.count == 1, "The sequence emitted more than one element.", file: file, line: line)
                         XCTAssertTrue(successCondition(emissions.first!), "Expectation violated - '\(description)'", file: file, line: line)
                         expect.fulfill()
-                    })
+                    }
+                }, receiveValue: {
+                    emissions.append($0)
+                 })
+                    .withLifetime(of: self)
```

Comment:
indent off?

Reply:
I ran swiftformat to fix the indent on line 31, line 32 is just how Xcode prefers it